ebml/EbmlElement.h

```diff
@@ -633,10 +633,8 @@ class EBML_DLL_API EbmlElementDefault : public EbmlElement {
 template<typename T>
 class EBML_DLL_API EbmlElementDefaultSameStorage : public EbmlElementDefault<T> {
   public:
+    using EbmlElementDefault<T>::EbmlElementDefault;
```

Comment:
Typo in the commit log. Also don't use "also" to refer to another commit in a commit title.
This and the previous commit(s) should be in a separate PR.

Reply:
Right.

Reply:
You still kept the "also" in the commit title in the other PR.

Reply:
Yeah, I didn't push my branch.

On this PR, I rebase against the other ones.